src/explorer/AzureAccountTreeItem.ts

```diff
@@ -25,8 +26,14 @@ export class AzureAccountTreeItem extends AzureAccountTreeItemBase {
     public async loadMoreChildrenImpl(clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
         const ti: AzExtTreeItem | undefined = this.trialAppNode ?? await this.loadTrialAppNode();
         const children: AzExtTreeItem[] = await super.loadMoreChildrenImpl(clearCache, context);
+
         if (ti) {
             children.push(ti);
+        } else {
+            // Must check children[0] instanceof GenericTreeItem because if Azure account is still loading then isLoggedIn is false.
+            if (!this.isLoggedIn && children[0] instanceof GenericTreeItem) {
+                children.push(new GenericTreeItem(this, { label: 'Create free trial app...', contextValue: 'azureCommand', commandId: `${ext.prefix}.CreateTrialApp`, iconPath: getIconPath('WebApp'), includeInTreeItemPicker: true }));
```

Comment:
Including it in the treePicker has some weird behaviors.  I think because it doesn't return anything, if I click on it from the command palette, it opens up the page to create, but doesn't close the quickpick or anything.